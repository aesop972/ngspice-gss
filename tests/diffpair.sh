#! /bin/sh

. $srcdir/config.sh
spicetest diffpair
