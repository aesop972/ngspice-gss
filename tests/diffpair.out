
Circuit: simple differential pair - CM and DM dc sensitivity

Transfer function information:
transfer_function = -1.10340e-01
output_impedance_at_v(5) = 9.446401e+03
vcm#input_impedance = 1.792489e+06

Transfer function information:
transfer_function = -8.79002e+01
output_impedance_at_v(5) = 9.446401e+03
vdm#input_impedance = 8.934860e+03




